src/poliastro/bodies.py

```diff
@@ -5,9 +5,27 @@
 
 * Sun (☉)
 * Earth (♁)
+* Moon (☾)
+* Mercury (☿)
+* Venus (♀)
+* Mars (♂)
+* Jupiter (♃)
+* Saturn (♄)
+* Uranus (⛢)
+* Neptune (♆)
+* Pluto (♇)
```

Comment:
Good catch!! I demand a blank line separating the Sun from the planets and another one separating Pluto from the planets then :smiling_imp: